src/main/java/com/jcabi/aether/RootArtifact.java

```diff
@@ -121,6 +129,47 @@ public Artifact artifact() {
     }
 
     /**
+     * Downloads parent artifact from repositories.
+     * @param system The Repository System
+     * @param session The Repository Session
+     * @param repos Repositories to send request
+     * @return Parent artifact
+     * @throws Exception if downloading fails
+     */
+    public Artifact lookUpParent(final RepositorySystem system,
+        final RepositorySystemSession session,
+        final List<RemoteRepository> repos)
+        throws Exception {
+        Artifact result = null;
+        final Parent parent = new DefaultModelBuilder()
+            .build(new DefaultModelBuildingRequest()
+                .setPomFile(this.art.getFile())
+                .setTwoPhaseBuilding(false)
+                .setSystemProperties(System.getProperties())
+                .setValidationLevel(
+                      ModelBuildingRequest.VALIDATION_LEVEL_MINIMAL
+                )
+        )
+            .getRawModel()
+            .getParent();
+        if (parent != null) {
+            result = system.resolveArtifact(
+                session,
+                new ArtifactRequest()
+                    .setArtifact(
+                        new DefaultArtifact(
+                            parent.getGroupId(),
+                            parent.getArtifactId(),
+                            "pom",
+                            parent.getVersion()
+                        )
+                ).setRepositories(repos)
+            ).getArtifact();
```

Comment:
@piddubnyi please format fluent calls according to http://www.yegor256.com/2014/10/23/paired-brackets-notation.html#fluent